generators/generator-bot-enterprise-assistant/README.md

```diff
@@ -13,29 +61,37 @@ npm install -g @microsoft/generator-bot-enterprise-assistant
 Then generate your new project:
 
 ```bash
-yo @microsoft/bot-assistant-core '{BOT_NAME}'
+yo @microsoft/bot-conversational-core '{BOT_NAME}' -platform '{dotnet|js}' -integration '{functions|webapp}'
```

Comment:
Update to reference enterprise-assistant, not conversational-core.

Reply:
fixed